File: solutions/java/1732-find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int max_altitude = 0;
        int current_altitude = 0;
        for(int g : gain){
            current_altitude += g;
            max_altitude = Math.max(current_altitude, max_altitude);
        }
        return max_altitude;
    }
}

File: solutions/go/1732-find-the-highest-altitude.go
func largestAltitude(gain []int) int {
    maxAltitude := 0 
    currentAltitude := 0 

    for _, g := range gain { 
        currentAltitude += g 
        if currentAltitude > maxAltitude { 
            maxAltitude = currentAltitude 
        } 
    } 
    return maxAltitude 
                                                               
}
